File: PHP/UberVan.php
<?php

class UberPool extends Car
{
  public $typeCarAccepted;
  public $seatsMaterial;

  public function __construct($license, $driver, $typeCarAccepted, $seatsMaterial)
  {
    // pasando los parametros a la superclase como en java super aca seria parent
    parent::__construct($license, $driver);

    $this->typeCarAccepted = $typeCarAccepted;
    $this->seatsMaterial = $seatsMaterial;
  }
}

File: Java/UberVan.java
import java.util.ArrayList;
import java.util.Map;

class UberVan extends Car {

    // map:representa la marca array: representara el modelo
    Map<String, Map<String, Integer>> typeCarAccepted;

    // leather or vinyl
    ArrayList<String> seatsMaterial;

    public UberVan(
            String license, Account driver,
            Map<String, Map<String, Integer>> typeCarAccepted,
            ArrayList<String> seatsMaterial) {

        super(license, driver);
        this.typeCarAccepted = typeCarAccepted;
        this.seatsMaterial = seatsMaterial;
    }

}
